Automattic/wp-calypso
# Theme Showcase: Unify styles and colors on LOTS & LITS

Related to https://github.com/Automattic/wp-calypso/issues/80626

## Proposed Changes

* Unified the colors of the different components and states on the LITS according to the `ResponsiveToolbarGroup`.
* Unified the colors of the different components and states on the LOTS according to the `ResponsiveToolbarGroup`.

|Before|After|
|----|----|
|![Captura de Pantalla 2023-08-18 a las 15 33 18](https://github.com/Automattic/wp-calypso/assets/1989914/fcc93712-4968-4c97-a942-f28e2ee7f638)|![Captura de Pantalla 2023-08-18 a las 15 35 11](https://github.com/Automattic/wp-calypso/assets/1989914/7a5dbb95-aa18-4e2f-bbfc-067cfa1225a6)|


## Testing Instructions



* Open the Livelink
* Navigate to the LITS
* The colors of the different elements should be consistent when selected, hovering, etc.
* Navigate to the LOTS
* The colors of the different elements should be consistent when selected, hovering, etc.

## Pre-merge Checklist
